lib/task-data/tasks/install-esx.js

```diff
@@ -4,7 +4,7 @@
 
 module.exports = {
     friendlyName: 'Install Esx',
-    injectableName: 'Task.Os.Install.Esx',
+    injectableName: 'Task.Os.Install.ESXi',
```

Comment:
they're not intended to be included in this PR, these PRs are only for user's interfaces,  not for code details. suggest to refine them during developing bit by bit.